pom.xml

```diff
@@ -1950,11 +1950,11 @@
                 <scope>compile</scope>
             </dependency>
 
-            <dependency>
-              <groupId>com.sleepycat</groupId>
-              <artifactId>je</artifactId>
-              <version>${sleepycat.version}</version>
-            </dependency>
+<!--            <dependency>-->
```

Comment:
Why is this commented?

Reply:
Agreed.... this is likely causing your build failure as you are looking for an older version of sleepycat which is not in the public repos.

Reply:
This will affect other components too -- can you confirm what version of sleepycat are used after the pr using
'mvn dependency:tree' and look for all instances of sleepycat. I would expect them to be at ${sleepycat.version}.

We need this dependency to uplift the version of the dependency transitively, which also fixes a build problem seen outside of our official/local builds, due to missing artifacts on maven central.

Reply:
Ah looking at latest changes I see this is now corrected. thanks :-)